Scaffolding.sln

```diff
@@ -616,6 +598,24 @@ Global
 		{8A75336B-A55F-42F6-B8BA-B55EFDA39705}.Release|x64.Build.0 = Release|Any CPU
 		{8A75336B-A55F-42F6-B8BA-B55EFDA39705}.Release|x86.ActiveCfg = Release|Any CPU
 		{8A75336B-A55F-42F6-B8BA-B55EFDA39705}.Release|x86.Build.0 = Release|Any CPU
+		{0294A050-806F-4FF5-989E-9755DF3D56B1}.debug_x86|Any CPU.ActiveCfg = Debug|Any CPU
+		{0294A050-806F-4FF5-989E-9755DF3D56B1}.debug_x86|Any CPU.Build.0 = Debug|Any CPU
+		{0294A050-806F-4FF5-989E-9755DF3D56B1}.debug_x86|x64.ActiveCfg = Debug|Any CPU
```

Comment:
Why is there a debug_x86|x64 configuration? Typically debug and platform are separate.

Reply:
VS did this automatically. Will clean it up.